File: testdata/hallucinated_deps/fake_imports.py
# Test file with hallucinated Python dependencies

import os  # stdlib - should be ignored
import json  # stdlib - should be ignored
from pathlib import Path  # stdlib - should be ignored

import requests  # real package - should pass
import flask  # real package - should pass

# These are fake packages that should be flagged
import nonexistent_ai_generated_package_12345
from fake_utils_library_xyz import helper
import totally_made_up_sdk

def main():
    print("Hello, world!")

if __name__ == "__main__":
    main()

File: testdata/hallucinated_deps/fake_imports.rs
// Test file with hallucinated Rust dependencies

use std::collections::HashMap;  // stdlib - should be ignored
use std::io::{self, Read};  // stdlib - should be ignored

use serde::{Deserialize, Serialize};  // real crate - should pass
use tokio::runtime::Runtime;  // real crate - should pass

// These are fake crates that should be flagged
use nonexistent_ai_generated_crate_12345;
use fake_utils_library_xyz::helper;
use totally_made_up_sdk;

fn main() {
    println!("Hello, world!");
}
